spec/config/initializers/ab_tests_spec.rb

```diff
@@ -261,4 +261,44 @@
       end
     end
   end
+
+  describe 'SOCURE_IDV_SHADOW_MODE' do
+    let(:user) { create(:user) }
+
+    subject(:bucket) do
+      AbTests::SOCURE_IDV_SHADOW_MODE.bucket(
+        request: nil,
+        service_provider: nil,
+        session: nil,
+        user:,
```

Comment:
Idle blather: I generally like the hash value omission feature, but it feels almost backwards in this usage, where the only value I _don't_ specify is the one that I have available and that is not nil.